From dc8cd1c43edeedb9f7335020537c4ffdddd683f8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Draszik?= <andre.draszik@jci.com>
Date: Thu, 7 Feb 2019 01:26:53 +0000
Subject: [PATCH 1/3] sock_server: fix compilation against musl (sys/errno.h)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Compiling against musl-libc fails with the following error:

| In file included from .../host/xtest/sock_server.c:24:
| .../usr/include/sys/errno.h:1:2: error: #warning redirecting incorrect #include <sys/errno.h> to <errno.h> [-Werror=cpp]
|  #warning redirecting incorrect #include <sys/errno.h> to <errno.h>
|   ^~~~~~~

Just remove the needless include.

Signed-off-by: André Draszik <andre.draszik@jci.com>
Acked-by: Jens Wiklander <jens.wiklander@linaro.org>
Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>
---
Signed-off-by: André Draszik <andre.draszik@jci.com>
Upstream-Status: Backport [3.5.0]
 host/xtest/sock_server.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/host/xtest/sock_server.c b/host/xtest/sock_server.c
index 0d2ff06..1ba73d6 100644
--- a/host/xtest/sock_server.c
+++ b/host/xtest/sock_server.c
@@ -21,7 +21,6 @@
 #include <netdb.h>
 #include <netinet/in.h>
 #include <poll.h>
-#include <sys/errno.h>
 #include <sys/socket.h>
 #include <unistd.h>
 
-- 
2.23.0.rc1

